sdks/go/pkg/beam/windowing.go

```diff
@@ -21,21 +21,47 @@ import (
 	"github.com/apache/beam/sdks/go/pkg/beam/internal/errors"
 )
 
+type WindowIntoOption interface {
+	private()
```

Comment:
"private" isn't a good name for this blocking function, because beam.Option is already using it. You can test this out yourself by seeing that you can pass `WindowIntoOption`s as an Option into beam.ParDo.

A better unexported method name would be `windowIntoOption()` to avoid this conflation.